### PR TITLE
Add pre-submit checks to see if plugin versions have been incremented correctly

### PR DESCRIPTION
**Pull request by iskakaushik**
Addresses https://github.com/flutter/flutter/issues/29815

### CODE REVIEW

File: script/version_checker/lib/main.dart [hunk 1]
```diff
@@ -0,0 +1,82 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```
Comment (amirh) [inline script/version_checker/lib/main.dart]: What do you think about putting this in the flutter_plugin_tools?

Comment (iskakaushik) [reply script/version_checker/lib/main.dart]: Ah, did not realize that it was a thing. Makes more sense for this to live there. Will add it.

**Comment by iskakaushik** [comment]
Will move this to: https://github.com/flutter/plugin_tools/tree/master/lib/src

**Comment by iskakaushik** [comment]
moved to https://github.com/flutter/plugin_tools/pull/34/files

**Pull request closed without merging**